metadata/metadata.go

```diff
@@ -0,0 +1,72 @@
+package metadata
```

Comment:
Why is this package located outside of pkg/?

Reply:
it was miss click, thanks, fixed.